theme/templates/base.html

```diff
@@ -196,7 +196,7 @@
                         <h3>{{ siteconf.col1_heading }}</h3>
                     </div>
                     <div class="content">
-                        {{ siteconf.col1_content|richtext_filters|safe }}
+                        {{ siteconf.col1_content|richtext_filters }}
```

Comment:
Yes, double check permissions for that stuff to make sure that only site admins can edit siteconf fields and then revert to |safe if that's true.